projects/aca-content/src/lib/dialogs/folder-details/folder-information.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="aca-folder-info-container">
+  <div class="aca-folder-info-header">
+    <div class="aca-folder-icon">
```

Comment:
I don't see styles for aca-folder-icon class. If it's added for testing purpose then data-automation-id should be used instead

Reply:
Removed unneeded div